platform/src/main/java/org/stellar/anchor/platform/service/TransactionService.java

```diff
@@ -91,6 +93,8 @@ public PatchTransactionsResponse patchTransactions(PatchTransactionsRequest requ
     for (JdbcSep31Transaction txn : txnsToSave) {
       // TODO: consider 2-phase commit DB transaction management.
       txnStore.save(txn);
+      Metrics.counter("sep31.transaction",
+              "status",txn.getStatus()).increment();
```

Comment:
‼️ This is not right. We cannot assume that all calls to `patchTransactions` are meant to update a specific status.

Firstly, you need to check if we're really updating a status here.

Also, to be accurate, you'll need to gather both the old and new statuses, so you can decrement the old status count and increment the new one.

Reply:
we cant decrement counters. @lijamie98 @reecexlm  and i had a call on this the other day and decided to just increment a counter every time a transaction transitions into a different state. this can be useful on a time series graph if we see the number of transactions in `complete` vs 'pending_*' starting to diverge too much. other alerts can also be build arount this. 

i'll add a check for status changes